### DISCUSSION TITLE
How to create desktop icon after build from suorce on ubuntu?

### DISCUSSION THREAD

**Discussion by TavakoliAli** [keepassxreboot/keepassxc#6897] [Q&A] [answered]
Hello
I follow the wiki instruction to build the Keepassxc v2.6.6 from source. It is built and installed successfully and i can run it from CLI but there isn't any desktop icon for it. How can i create that?

Thanks

**Comment by droidmonkey** [comment, marked as answer]
This might help: https://www.maketecheasier.com/create-desktop-shortcuts-ubuntu/
